tests/data/test_package/test_package/test_package.py

```diff
@@ -0,0 +1,8 @@
+class TestPackage:
```

Comment:
Is it intended that all parent folders and API elements are some variation of `TestPackage`? I find it rather hard to validate the snapshots with those names.

Reply:
Removed these snapshot tests and added unit tests instead.